impress-org/givewp-next-gen
# Feature: add focus ring for blocks

## Description



This PR adds a focus ring for the blocks when they are selected. This feature is helpful because provides UI feedback about which block is selected.

## Affects



The form builder

## Visuals



![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/b53c2797-4689-4ce2-9a2f-4e1901e5e488)

![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/76770842-9cdc-41f9-a4a5-79387093f92a)


## Testing Instructions



Click on any block in the form builder and you should see a blue focus ring indicating that the block is selected, check the attached screenshot for reference.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204927308757475
